CSE/Logic/Database/Reader.cs

```diff
@@ -6,6 +6,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using Logic.Metadata;
+using System.Diagnostics;
```

Comment:
In my opinion, debugging stuff should be left in your local machine. (Exclude System.Diagnostics and remove Debug statements, but later).